indico/modules/events/registration/client/js/form_submission/index.jsx

```diff
@@ -32,6 +32,7 @@ export default function setupRegformSubmission(root) {
     publishToParticipants,
     publishToPublic,
     consentToPublish = null,
+    captchaRequired = false,
```

Comment:
I'm surprised this works - i'd have expected `JSON.parse(false)` to fail or at cause log a warning...